scripts/dotnet-cli-build/PackageTargets.cs

```diff
@@ -97,6 +97,20 @@ public static BuildTargetResult CopySharedHostLayout(BuildTargetContext c)
                 var destFile = file.Replace(Dirs.Stage2, sharedHostRoot);
                 File.Copy(file, destFile, true);
             }
+            
+            string versionedFxrDir = Path.Combine(sharedHostRoot, "host", "fxr", c.BuildContext.Get<BuildVersion>("BuildVersion").NuGetVersion);
+            Directory.CreateDirectory(versionedFxrDir);
+            foreach (var file in Directory.GetFiles(Dirs.Stage2, "*", SearchOption.TopDirectoryOnly))
+            {
+                if (!Path.GetFileName(file).StartsWith("hostfxr"))
+                {
+                    continue;
+                }
+                
+                var destFile = file.Replace(Dirs.Stage2, versionedFxrDir);
+                File.Copy(file, destFile, true);
```

Comment:
Im repeating the existing logic. I think Copy is better since investigation of possible issues will be easier